share/completions/eopkg.fish

```diff
@@ -0,0 +1,153 @@
+function __fish_eopkg_print_components -d "Print list of components"
+    eopkg list-components -N | cut -d ' ' -f 1
+end
+
+function __fish_eopkg_print_repos -d "Print list of repositories"
+    eopkg list-repo -N | grep active | cut -d ' ' -f 1
+end
+
+# Setup additional completion
+complete -f -c eopkg -n '__fish_eopkg_package_ok; and not __fish_eopkg_component_ok' -a "(__fish_print_packages)" -d "package"
+complete -f -c eopkg -n '__fish_eopkg_component_ok' -a "(__fish_eopkg_print_components)" -d "component"
+complete -f -c eopkg -n '__fish_eopkg_repo_ok' -a "(__fish_eopkg_print_repos)" -d "repository"
+
+# Setup eopkg subcommand with shortcut
+__fish_eopkg_subcommand_with_shortcut upgrade up -d "Upgrades eopkg packages"
+__fish_eopkg_option_with_shortcut upgrade up -l security-only -d "Security related package upgrades only"
+__fish_eopkg_option_with_shortcut upgrade up -s n -l dry-run -d "Do not perform any action, just show what would be done"
+__fish_eopkg_option_with_shortcut upgrade up -s c -l component -f -d "Upgrade component's and recursive components' packages"
+
+__fish_eopkg_subcommand_with_shortcut install it -d "Install eopkg packages"
+__fish_eopkg_option_with_shortcut install it -l reinstall -d "Reinstall already installed packages"
+__fish_eopkg_option_with_shortcut install it -s n -l dry-run -d "Do not perform any action, just show what would be done"
+__fish_eopkg_option_with_shortcut install it -s c -l component -f -d "Install component's and recursive components' packages"
+
+__fish_eopkg_subcommand_with_shortcut remove rm -d "Remove eopkg packages"
+__fish_eopkg_option_with_shortcut remove rm -l purge -d "Removes everything including changed config files of the package"
+__fish_eopkg_option_with_shortcut remove rm -s n -l dry-run -d "Do not perform any action, just show what would be done"
+__fish_eopkg_option_with_shortcut remove rm -s c -l component -f -d "Remove component's and recursive components' packages"
+
+__fish_eopkg_subcommand_with_shortcut remove-orphans rmo -d "Remove orphaned packages"
+__fish_eopkg_option_with_shortcut remove-orphans rmo -l purge -d "Removes everything including changed config files of the package"
+__fish_eopkg_option_with_shortcut remove-orphans rmo -s n -l dry-run -d "Do not perform any action, just show what would be done"
+
+__fish_eopkg_subcommand_with_shortcut autoremove rmf -d "Remove packages and dependencies"
+__fish_eopkg_option_with_shortcut autoremove rmf -l purge -d "Removes everything including changed config files of the package"
+__fish_eopkg_option_with_shortcut autoremove rmf -s n -l dry-run -d "Do not perform any action, just show what would be done"
+
+__fish_eopkg_subcommand_with_shortcut history hs -d "History of eopkg operations"
+__fish_eopkg_option_with_shortcut history hs -s l -l last -x -d "Output only the last n operations"
+__fish_eopkg_option_with_shortcut history hs -s t -l takeback -x -d "Takeback to the state after the given operation finished"
```

Comment:
"finished" is kinda superfluous here, no? "Takeback to the state after the given operation"?